news/garage.rst

```diff
@@ -109,18 +109,21 @@ obvious reason.**
 NBE garage, F-building
 ----------------------
 
-We are piloting also a data & science garage at NBE every first
-wednesday of the month, room F336 13:00. NBE garage will also cover
+NBE garage used to happen in person every first
+wednesday of the month, room F336 13:00. NBE garage covers also
 issues related to working with personal data, research ethics, best
-practices in biomedical research.
+practices in biomedical research. 
+
+**Due to the current situation NBE garage
+happens online at the same time of the Aalto Sci Comp garage (see above).**
```

Comment:
Do you want to designate a special NBE day when we would specifically invite the other NBE/PHYS people you mentioned to come?

Reply:
The participation has been so weak that it's easier if they come at "Triton" garage. We will most likely start again the zoom NBE-ITS in August, so they also have that opportunity.